packages/client/package.json

```diff
@@ -87,7 +87,10 @@
   },
   "dependencies": {
     "@grpc/grpc-js": "^1.8.16",
+    "@protobuf-ts/grpc-transport": "^2.9.0",
+    "@protobuf-ts/grpcweb-transport": "^2.9.0",
```

Comment:
Is this package necessary also for "server" client? Can we move it into `packages/client-browser/package.json`?

Reply:
There's no dependencies in client-browser. All building and deciding which library to use is done by client and `client-browser/package.json` just contains metadata for package but all building steps are handled by tsup configuration in client

Reply:
Nice! 👍 Thanks for clarification